test/e2e/agent/recipes_test.go

```diff
@@ -138,7 +138,8 @@ func TestFleetKubernetesNonRootIntegrationRecipe(t *testing.T) {
 	}
 
 	if (v.GE(version.MinFor(9, 0, 1)) && v.LE(version.MinFor(9, 0, 4))) ||
-		(v.EQ(version.From(9, 1, 0))) {
+		(v.EQ(version.From(9, 1, 0))) ||
+		(v.EQ(version.MustParse("9.3.0-SNAPSHOT"))) { // TODO remove once https://github.com/elastic/kibana/pull/230211 is backported to 9.3.x
```

Comment:
nit: if this is supposed to be fixed before the next ECK release could we create an issue with the `v3.3.0 (next)` label?